src/explore-education-statistics-common/src/modules/table-tool/components/TableHeadersGroupControls.tsx

```diff
@@ -160,14 +161,23 @@ const TableHeadersGroupControls = ({
                   <VisuallyHidden>{` items in ${legend}`}</VisuallyHidden>
                 </Button>
                 <Button
-                  className="govuk-!-width-one-half"
+                  className="govuk-!-width-one-third"
                   disabled={!!disableControls}
                   onClick={() => {
                     toggleMoveControlsActive(id);
                   }}
                 >
                   Move<VisuallyHidden> {legend}</VisuallyHidden>
                 </Button>
+                <Button
```

Comment:
I think it might be better to show this button when you've clicked the 'reorder' button instead of here as buttons here relate to the group rather than the items in it.

Reply:
I had a chat with Cam about this and we have decided to keep it where it is at least for now, as the acceptance criteria wanted it where it currently is. If we decide down the line it is better elsewhere we can make the appropriate changes.